### DISCUSSION TITLE
nativeScrollbarsOverlaid for v2?

### DISCUSSION THREAD

**Discussion by digidux** [KingSora/OverlayScrollbars#473] [Q&A] [answered]
Hi! I'm trying to disable OS for mobile devices which already have native scrollbars. I'm currently using v2 beta 3.

`nativeScrollbarsOverlaid: { initialize: false }` doesn't work. I did some source diving and found it's been replaced by `nativeScrollbarsOverlaid`. But true and false doesn't work either. 

Is there another way?

**Comment by KingSora** [comment, marked as answer]
Good day @digidux 

The `showNativeOverlaidScrollbars` option is the replacement for `nativeScrollbarsOverlaid.show` in `v2`. 

The `nativeScrollbarsOverlaid.initialize` options is completely removed from the options in `v2` and replaced with the "initialization object". The idea is to fully customize the initialization process with it:

```js
OverlayScrollbars({ 
 target: document.querySelector('#target'),
 cancel: {
   nativeScrollbarsOverlaid: true,
 }
}, {});
```

You are looking for the `cancel` field there, which indicates when to cancel the initialization. Additionally to `cancel.nativeScrollbarsOverlaid` you can also specify `cancel.body`. You can read the typescript definitions for more info :)

**Reply by digidux** [reply]
Wow I must've missed that! Thanks! :)
